fastlane/lib/fastlane/actions/app_store_build_info.rb

```diff
@@ -0,0 +1,198 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      LATEST_BUILD_NUMBER = :LATEST_BUILD_NUMBER
+      LATEST_VERSION_NUMBER = :LATEST_VERSION_NUMBER
+    end
+
+    class AppStoreBuildInfoAction < Action
+      def self.run(params)
+        require 'spaceship'
+
+        build_nr = get_build_number(params)
+
+        # Convert build_nr to int (for legacy use) if no "." in string
+        if build_nr.kind_of?(String) && !build_nr.include?(".")
+          build_nr = build_nr.to_i
+        end
+
+        Actions.lane_context[SharedValues::LATEST_BUILD_NUMBER] = build_nr
+      end
+
+      def self.get_build_number(params)
+        UI.message("Login to iTunes Connect (#{params[:username]})")
+        Spaceship::Tunes.login(params[:username])
+        Spaceship::Tunes.select_team(team_id: params[:team_id], team_name: params[:team_name])
+        UI.message("Login successful")
+
+        platform = params[:platform]
+
+        app = Spaceship::Tunes::Application.find(params[:app_identifier])
+        if params[:live]
+          UI.message("Fetching the latest build number for live-version")
+          live_version = app.live_version
+          build_nr = live_version.current_build_number
+          Actions.lane_context[SharedValues::LATEST_VERSION_NUMBER] = live_version.version
```

Comment:
Also, `version_number` only gets set in the `else` in this if statement. We probably need to add a line in this first part of this app that is `version_number = live_version.version`

Reply:
added.